<ui version="4.0" >
 <class>TextEditor</class>
 <widget class="QWidget" name="TextEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QFontComboBox" name="fontComboBox" >
       <property name="toolTip" >
        <string>Set the Font Family</string>
       </property>
       <property name="currentIndex" >
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fontSizeSpinBox" >
       <property name="toolTip" >
        <string>Set the font size</string>
       </property>
       <property name="value" >
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="colorButton" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="boldButton" >
       <property name="font" >
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip" >
        <string>Toggle Bold Characters</string>
       </property>
       <property name="text" >
        <string>B</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italicButton" >
       <property name="toolTip" >
        <string>Toggle Italic Characters</string>
       </property>
       <property name="text" >
        <string>I</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="scriptButton" >
       <property name="toolTip" >
        <string>Set Superscript</string>
       </property>
       <property name="text" >
        <string>-</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="halButton" >
       <property name="text" >
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="specialCButton" >
       <property name="font" >
        <font>
         <family>Standard Symbols L</family>
        </font>
       </property>
       <property name="toolTip" >
        <string>Choose Special Character</string>
       </property>
       <property name="text" >
        <string>S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="imageSizeLabel" >
       <property name="text" >
        <string>Image size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="imageSizeSpinBox" >
       <property name="maximum" >
        <number>200</number>
       </property>
       <property name="minimum" >
        <number>10</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>31</width>
         <height>36</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textEdit" >
     <property name="readOnly" >
      <bool>false</bool>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
